src/main/java/net/datafaker/providers/videogame/FinalFantasyXIV.java

```diff
@@ -0,0 +1,37 @@
+package net.datafaker.providers.videogame;
```

Comment:
Should we move other video games in here?
Should this be a sub-package within entertainment?

@bodiam

Reply:
Whatever we do, it will never be right. Lord of the Rings. Is that a book, movies, a game, tv series?

But I am not really bothered with how it's organized. If there's some consistency, that's nice, and if you want to move don't packages around, I won't stop you :)

Reply:
Alright,that's fair.